### ISSUE TITLE
Stackblitz demo and local install not working: Error: __filename is not defined

### ISSUE THREAD

**Issue by simple10** [unplugin/unplugin-turbo-console#41]
### Describe the bug

I tried the official [Stackblitz demo](https://stackblitz.com/github/yuyinws/stackblitz-demo?file=README.md) as well as a local install with sveltekit. Both generate the same error.

Is there a quick fix for this? Maybe vite updated and broke trubo-console?

failed to load config from /home/projects/yqnimojwor.github/vite.config.ts
error when starting dev server:
Error: __filename is not defined
    at async loadConfigFromFile (file:///home/projects/yqnimojwor.github/node_modules/.pnpm/vite@5.3.3_@types+node@20.14.10/node_modules/vite/dist/node/chunks/dep-CzJTQ5q7.js:66874:24)
    at async resolveConfig (file:///home/projects/yqnimojwor.github/node_modules/.pnpm/vite@5.3.3_@types+node@20.14.10/node_modules/vite/dist/node/chunks/dep-CzJTQ5q7.js:66484:24)
    at async _createServer (file:///home/projects/yqnimojwor.github/node_modules/.pnpm/vite@5.3.3_@types+node@20.14.10/node_modules/vite/dist/node/chunks/dep-CzJTQ5q7.js:63097:18)
 ELIFECYCLE  Command failed with exit code 1.

### Reproduction

Reproduction

### System Info

```Shell
See [Stackblitz demo](https://stackblitz.com/github/yuyinws/stackblitz-demo?file=README.md)
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by simple10** [comment]
I just test v1.9.6 in local dev and it seems to work. Bug was introduced after v1.9.6 when __filename support was added.

**Comment by yuyinws** [comment]
Thanks for your report and test, I just fix it on v1.9.9. 
Local dev should been works.
stackblitz online demo still have some problem, I will check it later.

**Comment by simple10** [comment]
Awesome! Thank you for the quick fix. v1.9.9 is working for me with sveltekit on local dev.
